tests/test_projectedLikelihood.py

```diff
@@ -81,21 +81,20 @@ def setUp(self):
         self.model.readEllipses(ev, self.nonlinear, self.fixed)
         self.amplitudes = numpy.zeros(self.model.getAmplitudeDim(), dtype=lsst.meas.modelfit.Scalar)
         self.amplitudes[:] = self.flux
-        # setup ideal exposure0: uses fit Wcs and Calib, has delta function PSF
+        # setup ideal exposure0: uses fit Wcs and PhotoCalib, has delta function PSF
         scale0 = 0.2*lsst.afw.geom.arcseconds
         self.crpix0 = lsst.afw.geom.Point2D(0, 0)
         wcs0 = lsst.afw.geom.makeSkyWcs(crpix=self.crpix0,
                                         crval=self.position,
                                         cdMatrix=lsst.afw.geom.makeCdMatrix(scale=scale0))
-        calib0 = lsst.afw.image.Calib()
-        calib0.setFluxMag0(10000)
+        photoCalib0 = lsst.afw.image.PhotoCalib(10)
```

Comment:
Is this a change of units?

Reply:
Sort of: `PhotoCalib(10) != Calib(10000)`, for any definition of `==`. However, the actual value of the photometric calibration is not important in this test, just self-consistency.